TCutility/analysis/ts_vibration.py

```diff
@@ -0,0 +1,107 @@
+import numpy as np
+from TCutility.results import read
+from scm import plams
+
+def atom_distance(molecule: plams.Molecule, atom1: int, atom2: int) -> float:
+    res = 0.0
+    for i,j in zip(molecule[atom1],molecule[atom2]):
+        res += (i - j)**2
+    return np.sqrt(res)
+
+def avg_relative_bond_length_delta(base: plams.Molecule, pos: plams.Molecule,neg: plams.Molecule, a1: int, a2: int) -> float:
+    basedist = atom_distance(base, a1+1, a2+1)
+    x = abs((atom_distance(pos, a1+1, a2+1)/basedist)-1)
+    y = abs((atom_distance(neg, a1+1, a2+1)/basedist)-1)
+    return (x+y)/2
+
+def determine_ts_reactioncoordinate(calc_dir: str, bond_tolerance: float = 1.28, min_delta_dist: float = 0.1) -> np.ndarray:
+    '''Function to retrieve reaction coordinate from a given transitionstate, using the first imaginary frequency.
+
+    Args:
+        calc_dir: path pointing to the desired calculation.
+        bond_tolerance: parameter for plams.molecule.guess_bonds() function
+        min_delta_dist: minimum relative bond length change before qualifying as active atom
+
+    Returns:
+        Array containing all the obtained reaction coordinates. Reaction coordinate format is [active_atom1, active_atom2, sign]
+        Symmetry elements are ignored, by convention the atom labels are increasing (atom1 < atom2)
+    '''
+
+    data = read(calc_dir)
+    assert 'modes' in data.properties.vibrations, 'Vibrational data is required, but was not present in .rkf file'
+
+    outputmol = data.molecule.output
+
+    base = np.array(outputmol).reshape(-1,3)
+    tsimode = np.array(data.properties.vibrations.modes[0]).reshape(-1,3)
+
+    posvib = outputmol.copy()
+    posvib.from_array(base + tsimode)
+    posvib.guess_bonds(dmax=bond_tolerance)
+    negvib = outputmol.copy()
+    negvib.from_array(base - tsimode)
+    negvib.guess_bonds(dmax=bond_tolerance)
+
+    pairs = np.where(posvib.bond_matrix() - negvib.bond_matrix())
+    rc = np.array(
+                [   [a+1, b+1, np.sign(posvib.bond_matrix()[a][b] - negvib.bond_matrix()[a][b])]
+                    for a, b in np.c_[pairs[0], pairs[1]] 
+                    if a < b and avg_relative_bond_length_delta(outputmol, posvib, negvib, a, b) > min_delta_dist
+                ],  dtype=int)
+    return rc
+
+
+def validate_transitionstate(calc_dir: str, rcatoms = None, bond_tolerance: float = 1.28, min_delta_dist: float = 0.1) -> bool:
```

Comment:
Put a **kwargs in the function definition. Add a link :func:`determine_ts_reactioncoordinate` to the docstring

Reply:
Instead of checking if the provided RC is equal to the predicted RC, check if the provided RC is a member of the predicted RC